cmd/thanos/downsample.go

```diff
@@ -95,7 +95,13 @@ func runDownsample(
 	}()
 
 	metrics := newDownsampleMetrics(reg)
+
 	statusProber := prober.NewProber(comp, logger, prometheus.WrapRegistererWithPrefix("thanos_", reg))
+	// Initiate HTTP listener providing metrics endpoint and readiness/liveness probes.
```

Comment:
To make this PR a little bit smaller, could we revert these changes that are only moving code around?